### DISCUSSION TITLE
Achievements<title>

### DISCUSSION THREAD

**Discussion by alanhornbaker** [community/community#49143] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I need some achievements that are more flattering to a junior developer and feel more satisfying to achieve.  For example, the three achievements that I have are: {
1. Quickdraw - "closed a file within 6 minutes of opening"
2. Yolo - "Merged without a review"
3. Pull-Shark - "2nd pull request merged"
 };

But, I also created my own project idea, created a repository for it, and then invited my classmates to contribute to the project in an open-source fashion.  My invitation must have been persuasive (read: "valuable") because I received at least some engagement and contribution from my classmates.

I feel like that's not bad for being a 3-month-old developer.  Why are there achievements for stuff that is meaningless (I.E., I opened a file, looked at it, and said "not now" after 5 minutes.  What of it????"), and not achievements for stuff that promotes solid professional skills, like the activities described above?

**Comment by alanhornbaker** [comment]
Someone please explain to me why the amount of time you have a file open for matters more than generating ideas that solve common problems, setting a groundwork for those ideas, generating participation in those ideas,

**Reply by CodeMaster7000** [reply]
@alanhornbaker Could you please specify what achievement you are referring to? In summary, each GitHub achievement promotes a different quality that one should have (i.e. Galaxy Brain for helping the community, Pair Extraordinaire for working together), and some by nature would simply be exploitable or much easier to collect.

I would recommend Pair Extraordinaire and Galaxy Brain as 2 achievements flattering to developers. For Pair Extraordinaire, all you have to do is add a co-author to a commit on a pull request, while Galaxy Brain can be achieved by answering 2 discussions. They are relatively quick to collect, while the latter is very satisfying.

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
